chia/util/keychain.py

```diff
@@ -132,6 +143,104 @@ def get_private_key_user(user: str, index: int) -> str:
     return f"wallet-{user}-{index}"
 
 
+@final
+@streamable
+@dataclass(frozen=True)
+class KeyDataSecrets(Streamable):
+    mnemonic: List[str]
+    entropy: bytes
+    private_key: PrivateKey
+
+    def __post_init__(self) -> None:
+        # This is redundant if `from_*` methods are used but its to make sure there can't be an `KeyDataSecrets`
+        # instance with an attribute missmatch for calculated cached values. Should be ok since we don't handle a lot of
```

Comment:
Missmatch --> mismatch

```suggestion
        # instance with an attribute mismatch for calculated cached values. Should be ok since we don't handle a lot of
```